src/frontend/komponenter/Fagsak/Vedtak/OppsummeringVedtak.tsx

```diff
@@ -104,25 +110,42 @@ const OppsummeringVedtak: React.FunctionComponent<IVedtakProps> = ({ fagsak, åp
         }
     };
 
-    const visSubmitKnapp = !erLesevisning() && åpenBehandling?.status === BehandlingStatus.UTREDES;
+    const minst1PeriodeErBegrunnet = () => {
+        return (
+            (aktivVedtak?.utbetalingBegrunnelser.filter(
+                (utbetalingsbegrunnelse: IRestUtbetalingBegrunnelse) => {
+                    return (
+                        utbetalingsbegrunnelse.begrunnelseType &&
+                        utbetalingsbegrunnelse.vedtakBegrunnelse
```

Comment:
Trekke ut sjekk i "utbetalingsBegrunnelseFyltUt" eller noe, for å tydeliggjøre at utbetalingsbegrunnelsene på det aktive vedtaket består av en blanding utfylte og ikke?